app/views/layouts/_doctype.html.haml

```diff
@@ -0,0 +1 @@
+!!! 5
```

Comment:
I am no expert in HAML/HTML.  What is this new file for?
cc @dclarizio @martinpovolny